### DISCUSSION TITLE
[Question]What tool in Flet was used to make these tabs?

### DISCUSSION THREAD

**Discussion by g7AzaZLO** [flet-dev/flet#1781] [Q&A]
### Question

There's a program like [this](https://i.stack.imgur.com/uVxKN.png) [this](https://i.stack.imgur.com/CqXDz.png). It is written in Python, using the Flet library. What was used to make all these tabs-windows? I have looked through all the documentation and found nothing similar. All these tabs can be moved with the mouse.

I've read all the documentation but haven't found anything similar

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment]
They were not made using Flet. 😕

**Reply by g7AzaZLO** [reply]
Are you sure? The top panel is 1 in 1 like Flet's.

**Reply by ndonkoHenri** [reply]
I am pretty sure.
Have a look at the address bar, and notice `localhost...index.html`. 
The scrollbars are equally not Flet-like. 🤔

**Reply by g7AzaZLO** [reply]
Okay, do you have any idea what it might be written on?

**Reply by ndonkoHenri** [reply]
Nope.
